ui.tests/test-module/specs/wizard/wizard.locale.spec.js

```diff
@@ -0,0 +1,78 @@
+/*******************************************************************************
+ * Copyright 2024 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+const afConstants = require("../../libs/commons/formsConstants");
+const sitesSelectors = require("../../libs/commons/sitesSelectors");
+describe('Locale - Authoring Test', function () {
+    const changeLanguage = function (str) {
+        cy.visit('http://localhost:4502/aem/forms.html/content/dam/formsanddocuments');
+        cy.get('[data-foundation-toggleable-control-src="/mnt/overlay/granite/ui/content/shell/userproperties.html"]').click();
+        cy.get('[data-foundation-toggleable-control-src="/mnt/overlay/granite/ui/content/shell/userpreferences.html"]').click();
+        cy.get('[name="language"].coral-Form-field._coral-Dropdown').first().click();
```

Comment:
Please don't make use of coral private CSS classes, since the same test would fail on 650. try to use specific public CSS selector